scripts/local-dev.sh

```diff
@@ -30,11 +32,14 @@ usage() {
     echo "Options:"
     echo "  -n/-N NAMESPACE              Default namespace for pods (required)"
     echo "  -m/-M NAMESPACE              Llama Model namespace (optional, use if model is in different namespace)"
+    echo "  -c/-C CONFIG                 Model config source: 'local' or 'cluster' (default: cluster)"
```

Comment:
New param `-c/-C <config>` where config is either `cluster` (default) or `local`. This allows us to pass in the value from cli instead of being prompted when the script runs.